google-cloud-core/src/main/java/com/google/cloud/PlatformInformation.java

```diff
@@ -38,10 +39,7 @@ public static boolean isOnGAE() {
   }
 
   public static boolean isOnGAEStandard7() {
-    //edge case: when a Java 7 GAE app is deployed to DevServer running on J8, SPECIFICATION_VERSION will be "1.8",
-    //but RUNTIME_JETTY_LOGGER is unset
-    return isOnGAE() && SPECIFICATION_VERSION.equals("1.7")
-        || isOnGAE() && RUNTIME_JETTY_LOGGER == null;
+    return GAE_JAVA7;
   }
 
   public static boolean isOnGAEStandard8() {
```

Comment:
This is also an opportunity to cheaply simply GAE Standard Java 8 detection, too. :smile: ([Reference](https://github.com/GoogleCloudPlatform/appengine-plugins-core/issues/444).)

Reply:
Thank you for letting me know!